### PR TITLE
Feature/add icon burger button to close side left panel

### PR DESCRIPTION
**Pull request by rkmdCodes**
This PR is closign feature request #421

https://github.com/idurar/idurar-erp-crm/assets/113243168/8b835e53-ee69-4ef9-99c8-f0ed34e5fa05

### CODE REVIEW

File: frontend/src/modules/CrudModule/index.jsx [hunk 1]
```diff
@@ -110,8 +111,13 @@ function FixHeaderPanel({ config }) {
...
           ></Button>
         </Col>
-        <Col className="gutter-row" span={22}>
+        <Col
+          className="gutter-row"
+          style={{ display: 'flex', justifyContent: 'space-between' }}
+          span={22}
+        >
           <h1 style={{ fontSize: 20, marginBottom: 20 }}>{config.PANEL_TITLE}</h1>
+          <CloseOutlined onClick={collapsePanel} style={{ marginBottom: 20 }} />
```
Comment (salahlalami) [inline frontend/src/modules/CrudModule/index.jsx:120]: @rkmdCodes please replace close icon with <MenuOutlined />

**Review by salahlalami — CHANGES_REQUESTED** [review]

File: frontend/src/modules/CrudModule/index.jsx (hunk 1 above)
Comment (rkmdCodes) [reply frontend/src/modules/CrudModule/index.jsx:120]: hi @salahlalami sir
thank you for review,
can you please specify which icon i use to replace close icon

Comment (salahlalami) [reply frontend/src/modules/CrudModule/index.jsx:120]: @rkmdCodes use this one <MenuFoldOutlined />

Comment (polymahh) [reply frontend/src/modules/CrudModule/index.jsx:120]: @salahlalami the icon does not show do you mean <MenuOutlined />

Comment (salahlalami) [reply frontend/src/modules/CrudModule/index.jsx:120]: @polymahh  @rkmdCodes I mean this one  MenuFoldOutlined  ,

**Comment by rkmdCodes** [comment]
hi @salahlalami @polymahh 
Thanks for the review
I just updated the icon to MenuFoldIcon

Please review the changes

File: frontend/src/modules/CrudModule/index.jsx (hunk 1 above)
Comment (rkmdCodes) [reply frontend/src/modules/CrudModule/index.jsx]: hi @salahlalami @polymahh i just updated the icon

**Review by salahlalami — APPROVED** [review]

**Review by polymahh — APPROVED** [review]

**Pull request merged**